src/content/en/updates/2017/02/lighthouse-jan.md

```diff
@@ -0,0 +1,147 @@
+
+[Lighthouse](/web/tools/lighthouse/) is an
+[open-source](https://github.com/GoogleChrome/lighthouse), automated tool for
+improving the quality of your web apps. You can install it as a
+[Chrome Extension][crx] or run it as a Node command line tool. When you
+give Lighthouse a URL, it runs a barrage of tests against the page and then
+generates a report explaining how well the page did and indicating areas for
+improvement.
+{: .note }
+
+<img src="/web/progressive-web-apps/images/pwa-lighthouse.png"
+     class="lighthouse-logo attempt-right" alt="Lighthouse Logo">
+
+<style>
+.lighthouse-logo {
+  height: 150px;
+  width: auto;
+}
+</style>
+
+Today, we're happy to announce the [1.5 release](https://github.com/GoogleChrome/lighthouse/releases/tag/1.5.0) of Lighthouse, a huge release, with over **128 PRs**. Lighthouse 1.5 includes a bunch of big new features, audits, and the usual bug fixes. You can install it from npm (`npm i -g lighthouse`) or [download the extension](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk) from the Chrome Web Store.
+
+## New Audits
+
+The **CSS Usage Audit** reports the number of unused styles rules in your page
+and the cost/time savings of removing them:
+
+## ![CSS Usage Audit](../../images/2017/02/lighthouse/image_1.png)
+
+The **Optimized Images Audit** reports images that are unoptimized and the
+cost/time savings of optimizing them:
+
+![Optimized Images AudiT](../../images/2017/02/lighthouse/image_2.png)
+
+The **Responsive Images Audit** reports images that are too big and their
+potential cost/time savings of sizing them correctly for the given device:
+
+![Responsive Images Audit](../../images/2017/02/lighthouse/image_3.png)
+
+The **Deprecations and Interventions Audit** lists console warnings from Chrome
+if your page is using deprecated APIs or features that have
+[interventions](https://www.chromestatus.com/features#intervention):
+
+![Deprecations and Interventions Audit](../../images/2017/02/lighthouse/image_4.png)
+
+## Report changes
+
+As you've seen, we've focused on making the report less visually cluttered by
+adding tabular data, hiding extraneous help text, and added new features to make
+navigating the data, easier.
+
+### Emulation settings
+
+It's easy to forget what throttling and emulation settings were used for a
+particular Lighthouse run. Lighthouse reports now include the **runtime emulation settings**
+that were used to create the report; a [long time feature request](https://github.com/GoogleChrome/lighthouse/issues/568):
+
+![Emulation settings](../../images/2017/02/lighthouse/image_5.gif)
+
+### More useful trace data
+
+Lighthouse metrics like "First meaningful paint", "Time to Interactive", etc are
+mocked out as User Timing measures and injected back into the trace data saved with the `--save-assets` flag.
+
+If you use the  `--save-assets` flag when running Lighthouse, drop the trace into
```

Comment:
This doesn't read like a complete sentence to me... maybe reword to "If you use the --save-assets flag, you can now drop the trace into DevTools or open it in the DevTools Timeline Viewer"